app/jobs/scheduled/clean_up_uploads.rb

```diff
@@ -39,6 +39,10 @@ def execute(args)
           next if ReviewableQueuedPost.pending.where("payload->>'raw' LIKE '%#{upload.sha1}%' OR payload->>'raw' LIKE '%#{encoded_sha}%'").exists?
           next if Draft.where("data LIKE '%#{upload.sha1}%' OR data LIKE '%#{encoded_sha}%'").exists?
           next if ThemeSetting.where(data_type: ThemeSetting.types[:upload]).where("value LIKE ?", "%#{upload.sha1}%").exists?
+          if defined?(ChatMessage) &&
+              ChatMessage.where("message LIKE '%#{upload.sha1}%' OR message LIKE '%#{encoded_sha}%'").exists?
```

Comment:
Can you use proper parameters for these to avoid injection? It's weird that the above queries don't either. That seems bad!

Reply:
Maybe it's because you have to do interpolation anyways and it looks funky?
```
ChatMessage.where("message LIKE ? OR message LIKE ?", "%#{upload.sha1}%", "%#{encoded_sha}%").exists?
```

You can't do `ChatMessage.where("message LIKE '%?%' OR message LIKE '%?%', upload.sha1, encoded_sha).exists?`, because it adds the `'`'s and you get 
```
SELECT 1 AS one FROM "chat_messages" WHERE "chat_messages"."deleted_at" IS NULL AND (message LIKE '%'f5b99ab717b4b39c63bd2f4b9e66a31ac0a013c7'%' OR message LIKE '%'z3MDpRfWGupAJ715eDOdrpGE3xZ'%') LIMIT 1
ActiveRecord::StatementInvalid: PG::SyntaxError: ERROR:  syntax error at or near "f5b99ab717b4b39c63bd2f4b9e66a31ac0a013c7"
LINE 1: ...ssages"."deleted_at" IS NULL AND (message LIKE '%'f5b99ab717...
```

Reply:
Oh right it's the `%`! Makes sense.